tests/php/ORM/DataListTest.php

```diff
@@ -58,6 +58,8 @@ class DataListTest extends SapphireTest
     // Borrow the model from DataObjectTest
     protected static $fixture_file = 'DataObjectTest.yml';
 
+    protected $use_database = true;
```

Comment:
Why is this line added? It's not needed. "Note that this flag is overruled by the presence of a `$fixture_file`, which always forces a database build."

Reply:
Forgot to remove a bunch of rando debug code.